File: front/src/app/shared/models/invoice.ts
export interface Invoice {
  Code: string;
  ClientId: number;
  City: string;
  Total: number;
  SubTotal: number;
  VAT: number;
  Retention: number;
  CreationDate: Date;
  State: string;
  Paid: boolean;
  PaidDate: Date;
}

File: MonolegalAPI/MonolegalAPI/Models/Invoice.cs
using MongoDB.Bson;

namespace MonolegalAPI.Models
{
    public class Invoice
    {
        public ObjectId Id { get; set; }
        public string Code { get; set; }
        public int ClientId { get; set; }
        public string City { get; set; }
        public int Total { get; set; }
        public int SubTotal { get; set; }
        public int VAT { get; set; }
        public int Retention { get; set; }
        public DateTime CreationDate { get; set; }
        public string State { get; set; }
        public bool Paid { get; set; }
        public DateTime PaidDate { get; set; }

    }
}
